holidays/countries/ivory_coast.py

```diff
@@ -0,0 +1,128 @@
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+from gettext import gettext as tr
+
+from holidays import FEB
+from holidays.groups import (
+    ChristianHolidays,
+    IslamicHolidays,
+    InternationalHolidays,
+    StaticHolidays,
+)
+from holidays.observed_holiday_base import ObservedHolidayBase, SUN_TO_NEXT_MON
+
+
+class IvoryCoast(
+    ObservedHolidayBase, ChristianHolidays, IslamicHolidays, InternationalHolidays, StaticHolidays
+):
+    """Ivory Coast holidays.
+
+    References:
+        * <https://natlex.ilo.org/dyn/natlex2/r/natlex/fe/details?p3_isn=44374>
+        * <https://www.droit-afrique.com/uploads/RCI-Decret-1996-205-jours-feries.pdf>
+        * <https://fr.wikipedia.org/wiki/F%C3%AAtes_et_jours_f%C3%A9ri%C3%A9s_en_C%C3%B4te_d%27Ivoire>
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Ivory_Coast>
+        * <https://www.timeanddate.com/holidays/ivory-coast/>
+        * <https://holidayapi.com/countries/ci/>
+    """
+
+    country = "CI"
+    # %s (estimated).
+    estimated_label = tr("%s (estimé)")
+    # %s (observed).
+    observed_label = tr("%s (observé)")
+    # %s (observed, estimated).
+    observed_estimated_label = tr("%s (observé, estimé)")
+    # Ivory Coast gained independence in 1960.
+    start_year = 1960
+    default_language = "fr"
+    supported_languages = ("en_CI", "en_US", "fr")
+
+    def __init__(self, islamic_show_estimated: bool = True, *args, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChristianHolidays.__init__(self)
+        # the observed dates for the Ivory Coast's islamic holidays have been verified against
+        # local references (COSIM) and align with the default Umm al-Qura calculations.
+        IslamicHolidays.__init__(self, show_estimated=islamic_show_estimated)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, IvoryCoastStaticHolidays)
+        kwargs.setdefault("observed_rule", SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # Secular Holidays.
+
+        # New Year's Day.
+        self._add_new_years_day(tr("1er janvier"))
+
+        # Labor Day.
+        self._add_observed(self._add_labor_day(tr("Fête du travail")))
+
+        # Independence Day
+        self._add_observed(self._add_holiday_aug_7(tr("Fête Nationale")))
+
+        # Introduced in 1996. See details: https://en.wikipedia.org/wiki/C%C3%B4te_d%27Ivoire#History
```

Comment:
It's better to move this link to the class docstring as well, with a suitable comment.